cmd/libs/go2idl/parser/parse_test.go

```diff
@@ -128,7 +138,12 @@ type FooObject interface {
 }
 
 
-func proto.AFunc( proto.Object proto.Object ) proto.Frobber {}
+func proto.AFunc(proto.Object, proto.Object) (proto.Frobber) {}
+
+
+var proto.AVar proto.Frobber
```

Comment:
These declarations aren't valid go. I suggest doing the below to test all the cases.

```
var myVar proto.Frobber = proto.AVar
```
```